src/microbe_stage/ProcessStatistics.cs

```diff
@@ -71,13 +66,12 @@ public void RemoveUnused()
         }
     }
 
-    public SingleProcessStatistics GetAndMarkUsed(BioProcess forProcess)
+    public SingleProcessStatistics GetAndMarkUsed(TweakedProcess forProcess)
     {
 #if DEBUG
-        if (forProcess == null!)
+        if (forProcess.Process == null!)
             throw new ArgumentException("Invalid process marked used");
 #endif
-
```

Comment:
I think the blank line after a pragma looks much better than code being all bunched up.